infrastructure/Taskfile.yml

```diff
@@ -1104,7 +1104,7 @@ tasks:
           lagoon raw --raw "query allProjects {
             allProjects {
             name
-              environments(type: PRODUCTION) {
+              environments(type: DEVELOPMENT) {
```

Comment:
Is this change intentional?

The comment for the task still states "Gets the latest deployment for each *production* environment".

In general I also think it is more valuable for us to see deployments to production over development.